src/Campaigns/Repositories/CampaignRepository.php

```diff
@@ -38,6 +38,17 @@ public function getById(int $id)
             ->get();
     }
 
+    /**
+     * Get Campaign by Form ID
+     */
+    public function getByFormId(int $id)
```

Comment:
Let's add an `@unreleased` tag here. Also, I added a method with the same name on #7571 so we'll need to manage this conflict later.

Reply:
Resolved https://github.com/impress-org/givewp/pull/7569/commits/cb91501f43449a1ffadc47b7ead992bd08e1a890